.vitepress/constants/mainnet_versions.js

```diff
@@ -3,7 +3,7 @@ const mainnetVersions = Object.freeze({
   "app-latest-sha": "915847191e80d836f862eea2664949d9a240abea",
   "core-latest-tag": "v1.35.0-tm-v0.34.29",
   "core-latest-sha": "f95a434e674610b3a2c5e7d26bd87da69ccaa213",
-  "node-latest-tag": "v0.13.2",
+  "node-latest-tag": "v0.13.3",
   "node-latest-sha": "c1b41b0973e9d140b7651295e879d27ad47f42c4",
```

Comment:
this is not yet live on mainnet, also the sha needs to be updated, too. thanks for the PR!